src/ui/components/ArrowLink/stylesheet.css

```
@@ -19,7 +19,8 @@
 
 .svg {
   position: absolute;
-  top: 1rem;
+  top: 50%;
+  margin-top: -2.5rem;
```

This should move the arrow to the center of the x-height in all browsers:

```suggestion
  margin-top: -2.75rem;
```